model/inp/ww3_grid.inp

```diff
@@ -317,45 +317,30 @@ $   limitation and the GSE alleviation.
 $
 $ Unstructured grids ------------------------------------------------ $
 $   UNST parameters       : Namelist UNST
-$                           UGBCCFL    : Turns on/off (TRUE/FALSE) the computation of the CFL number on the physical domain boundary. 
-$                                        If FALSE the explicit scheme can be much faster though stability is not guaranteed
-$                                        (default TRUE)
-$                           UGOBCAUTO  : TRUE: OBC points are taken from  type 15 elements (default)
-$                                        FALSE: OBC points must be listed in ww3_grid.inp
-$                           UGOBCDEPTH : Threshold ( < 0) depth (default -10) for OBC points if UGOBCAUTO is TRUE
-$                           UGOBCFILE  : File name of file for reading boudary (default 'unset') 
-$
-$                           The following are TRUE/FALSE variables and only one can be TRUE. 
-$                           By default, EXPFSN is TRUE and must be set to false to activate another option  
-$                           EXPFSN    : Activation of N scheme (default option)
+$                           UGOBCAUTO : TRUE: OBC points are taken from  type 15 elements
```

Comment:
I'm not sure if the changes here are intentional.   It looks like this is removing descriptions of variables that should have descriptions, plus from below it seems like we should have a few additional variables added here.